examples/hello_libhttpserver.cpp

```diff
@@ -0,0 +1,30 @@
+/** ----------------------------------------------------------------------------
+ * @file    hello_libhttpserver.cpp
+ * @author  Sebastian Kaupper <kauppersebastian@gmail.com>
+ * @brief   Example program showcasing libhttpserver
+ *
+ * This program starts a server on port 8080 which listens to the endpoint /hello
+ * To verify that it's working use `url -XGET %http://localhost:8080/hello` or use
+ * a browser and go to <http://localhost:8080/hello>
```

Comment:
typo, think you meaned `curl` ^^ not `url`
think also the % is wrong

Reply:
true, it should indeed be `curl`
the % is used to prevent automatic link generation, so that it wont get a clickable link